contents/docs/session-replay/network-recording.mdx

```diff
@@ -85,7 +85,7 @@ Due the very high volume of network requests that some tools can make (for examp
 
 PostHog has to wrap `fetch` and `xhr` in order to capture network requests. If your application makes network requests before PostHog has had a chance to wrap them, then PostHog will not capture all information about the request.
 
-## PostHog truncated the request or response body
+## PostHog truncated the request or response body
```

Comment:
what has changed here?

do I have a broken space bar 🤣

Reply:
I really don't know tbh. I was just reading through the docs to see if anything needing updating per @andyvan-ph's request and found a single malformed title